### PR TITLE
refactors pureport.py module

### PR DESCRIPTION
**Pull request by privateip**
This commit refactors module_utils/pureport.py to
module_utils/pureport_client.py to help avoid namespace collisions in
some situations.

Signed-off-by: Peter Sprygada <peter.sprygada@pureport.com>

### CODE REVIEW

File: plugins/module_utils/pureport_client.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
     Client = None
     ClientHttpException = None
 
+API_URL="https://api.pureport.com"
```
Comment (mtraynham) [inline plugins/module_utils/pureport_client.py]: Likely want to remove this, the `API_URL` is imported above from the client.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,10 +33,10 @@ When importing a `module_util` use the full path to Python module instead of rel
 
 ```python
 # like this
-import ansible_collections.pureport.pureport.plugins.module_utils.pureport
+import ansible_collections.pureport.pureport.plugins.module_utils.pureport_client
```
Comment (mtraynham) [inline CONTRIBUTING.md]: Probably need to change these to reflect the Ansible 2.9 imports changes.

**Review by mtraynham — CHANGES_REQUESTED** [review]
Just 2 comments.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -33,10 +33,10 @@ When importing a `module_util` use the full path to Python module instead of rel
...
 
 # or like this
-from ansible_collections.pureport.pureport.plugins.module_utils.pureport import get_client
```
Comment (mtraynham) [inline CONTRIBUTING.md:39]: Also here.

File: plugins/module_utils/pureport_client.py (hunk 1 above)
Comment (privateip) [reply plugins/module_utils/pureport_client.py]: ugh good catch

**Review by mtraynham — APPROVED** [review]

**Pull request merged**